x-pack/plugins/translations/translations/ja-JP.json

```diff
@@ -8479,7 +8479,6 @@
     "xpack.reporting.exportTypes.csv.hitIterator.expectedScrollIdErrorMessage": "次の Elasticsearch からの応答で期待される {scrollId}: {response}",
     "xpack.reporting.exportTypes.png.compShim.failedToDecryptReportJobDataErrorMessage": "レポートジョブデータの解読に失敗しました{encryptionKey} が設定されていることを確認してこのレポートを再生成してください。{err}",
     "xpack.reporting.exportTypes.printablePdf.compShim.failedToDecryptReportJobDataErrorMessage": "レポートジョブデータの解読に失敗しました{encryptionKey} が設定されていることを確認してこのレポートを再生成してください。{err}",
-    "xpack.reporting.exportTypes.printablePdf.compShim.unableToGenerateReportErrorMessage": "URL {savedObjUrl} のレポートを生成できません。Kibana URL ではありません。",
```

Comment:
Are these deletion intentional? I couldn't find anything related to `xpack.reporting.exportTypes.printablePdf.compShim.unableToGenerateReportErrorMessage` here, but I'm assuming it's due to the underlying files themselves being deleted.

Reply:
It was the message that said, "Can't run the report: Invalid Kibana URL!". It's hard to find in the diff because the entire file was deleted: the compatibility_shim for execute_job.

That's not really possible from the code anymore, because execute_job doesn't handle URLs that are absolute to begin with: it always starts out with relative URLs.